### ISSUE TITLE
How can i send bundle or transaction in devnet?

### ISSUE THREAD

**Issue by datnguyen-tech** [jito-labs/jito-ts-rpc#3]
I want to test send tx on devnet, and I try change url of testnet Block Engines to **https://dallas.testnet.block-engine.jito.wtf,** and change url rpc to **devnet**.
But it not working, please help me!

![CleanShot 2024-09-18 at 17 49 22@2x](https://github.com/user-attachments/assets/6e6b0fd3-2068-4bfa-9217-69db017638c9)

![CleanShot 2024-09-18 at 17 51 27@2x](https://github.com/user-attachments/assets/50a62d13-61e4-4e50-843f-7bb90343bfa5)

**Comment by amalnathsathyan** [comment]
Yeah, I have the same doubt. I wanted to try a bundler buy on Raydium Devnet. But the blockengines are available on Testnets. Any work arounds ?

**Comment by 21e8** [comment]
don't think there is a workaround, pal. unless they release devnet endpoints but so far there are none

**Comment by LmingXie** [comment]
Are there plans to deploy a devnet Jito endpoint?

**Comment by fastparrotlol** [comment]
when you're gonna deploy it on devnet?

**Comment by VivekKadvani** [comment]
waiting...

**Comment by xmrbk** [comment]
Cho tôi xin địa chỉ ví tip jito testnet được không bro? @datnguyen-tech

**Comment by datnguyen-tech** [comment]
> Cho tôi xin địa chỉ ví tip jito testnet được không bro? @datnguyen-tech

Tôi đoán ví tip test/devnet chính là địa chỉ ví mainnet đó @xmrbk

**Comment by xmrbk** [comment]
> > Cho tôi xin địa chỉ ví tip jito testnet được không bro? @datnguyen-tech
> 
> Tôi đoán ví tip test/devnet chính là địa chỉ ví mainnet đó @xmrbk

dùng cái curl lấy ví tip của nó là được bro ơi.
ví tip testnet đây: 84DrGKhycCUGfLzw8hXsUYX9SnWdh2wW3ozsTPrC5xyg

**Comment by mdr0id** [comment]
We are working to get devnet with the same working endpoints. Hope to provide updates soon

**Comment by BrothersWar** [comment]
> We are working to get devnet with the same working endpoints. Hope to provide updates soon

hi! any updates?

**Comment by CodingFu** [comment]
@mdr0id any updates?
